demisto_sdk/commands/validate/file_validator.py

```diff
@@ -720,7 +720,7 @@ def is_valid_structure(self):
         if self.use_git:
             if self.branch_name != 'master' and (not self.branch_name.startswith('19.') and
                                                  not self.branch_name.startswith('20.')):
-                print('Validates only committed files')
+                print('Validates changed files')
                 self.validate_committed_files()
```

Comment:
Does `self.validate_committed_files()` validate _all_ changed files, not just committed ones? Cause it seems odd that you would update the `print` message here if it only validates committed files... If it validates all changed files the method name should probably be updated to reflect the true behavior of the method and not just the print statement above the method invocation.

Reply:
`validate_committed_files` checks files according to the content of `--post-commit` files
So just saying that it checks committed files is wrong.
I could add a print according to --post-commit if that makes more sense to you